packages/uni_app/lib/model/entities/library_occupation.dart

```diff
@@ -37,6 +37,10 @@ class LibraryOccupation {
     return floors[number - 1];
   }
 
+  void sortFloors() {
+    floors.sort((a, b) => a.number.compareTo(b.number));
+  }
+
```

Comment:
This is not relevant to this PR, right?
If so, please exclude these changes from the PR.